SalesforceSDK/Salesforce.Sample.HybridMainPage.Universal/ApplicationInsights.config

```diff
@@ -0,0 +1,23 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Is this is a Windows10 specific file? If yes then lets make sure we dont add anything that is specific to Windows 10 to the solution. Users can add it manually via Add->Exisiting Project.

@smcnulty-sfdc What are your thoughts on this?